docs/products/opensearch/howto/control_access_to_content.rst

```diff
@@ -0,0 +1,34 @@
+Control access to content in your service
+=========================================
+
+This article shows you how to set up access control lists (ACL) for content in Aiven for OpenSearch services.
+
+See the :doc:`related documentation <../concepts/access_control>` for more information on the permissions and patterns that are supported in ACL rules.
+
+To define ACLs for your service:
+
+1. Log in to the Aiven web console and select your OpenSearch service.
+
+   
+2. Click the **ACL** tab.
+
+3. Switch on **Enable ACL**.
+
+   This creates a set of rules that grant full access (``_*/admin``, ``*/admin``) for each existing service user.
+
+   .. note::
+      After you switch on ACL support, the service does not automatically add an ACL for new service users that you add. By default, access is denied.
+
+   
+4. If you want to enforce index rules in a limited fashion for requests that use the ``_mget``, ``_msearch``, and ``_bulk`` APIs for this service user, switch on **Enable extended ACLs**.
+   
+5. To define new ACLs:
```

Comment:
This is almost a different task. I think they should stay together since they're both quite short but let's have something like:
 - one section on enabling ACLs or extended ACLs as a task
 - and then, another section how to create each individual ACL as another task, this one will be repeated, the other one only needs to be done once.

Reply:
I'm not sure about this - I understand the use case for adding and editing ACLs when the feature is already in use, but is there a use case for just switching on ACLs without defining new ones or editing any existing ones?

Reply:
I don't think there's a case for switching on and not creating at least one ACL. But I think there are many cases where it's enabled and new ACLs would be created, or existing ones edited. And in that case, the task could be separate.

Reply:
I think we could have a separate task in the same topic (two sub-tasks with their own headings) - as these are very related and rather small. Also, as creating user ACLs has 7 steps on its own, maybe being separate, that looked a bit more approachable and easy.

Reply:
OK - that's now updated ;)